controller/service.go

```diff
@@ -234,6 +241,14 @@ func (c *controller) isServiceAllocated(key string) bool {
 	return c.ips.Pool(key) != ""
 }
 
+func (c *controller) allocatedIPs(svc string) []net.IP {
+	return c.ips.IPs(svc)
+}
+
+func (c *controller) isIPAssignable(ips []net.IP) bool {
```

Comment:
correct, I should use another word instead of assignable. But the idea is to verify that the ip is still part of a pool. Will delete the wrapper. I wanted to have something similar to the method 'isServiceAllocated' but it's not really needed.